recipes/xnvme-sys/meta.yaml

```diff
@@ -0,0 +1,74 @@
+---
+{% set name = "xnvme" %}
+{% set version = "0.7.4" %}
+{% set sha256 = "6d42d0bd7e6b395a37869a8713d138d332500f20ecc272ebf6789471026a7191" %}
+
+package:
+  name: {{ name|lower }}-sys
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/releases/download/v{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: true  # [not linux]
+  script: |
```

Comment:
For better readbility, put this into `build.sh`

Reply:
ack. I also put the tests into run_test.sh.